documentation/modules/con-entity-operator-logging.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-entity-operator.adoc
+
+[id='con-entity-operator-logging-{context}']
+= Operator loggers
+
+The Topic Operator and User Operator have a configurable logger:
+
+* `rootLogger.level`
+
+The operators use the Apache `log4j2` logger implementation.
+
+Use the `logging` property to configure loggers and logger levels.
+
+You can set the log levels by specifying the logger and level directly (inline) or use a custom (external) ConfigMap.
+If a ConfigMap is used, it must be placed under under the `log4j2.properties` key.
+
+Here we see examples of `inline` and `external` logging.
+
+.Inline logging
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    # ...
+  zookeeper:
+    # ...
+  entityOperator:
+    # ...
+    topicOperator:
+      watchedNamespace: my-topic-namespace
+      reconciliationIntervalSeconds: 60
+      logging:
+        type: inline
+        loggers:
+          rootLogger.level: INFO
+    # ...
+    userOperator:
+      watchedNamespace: my-topic-namespace
+      reconciliationIntervalSeconds: 60
+      logging:
+        type: inline
+        loggers:
+          rootLogger.level: INFO
+# ...
+----
+
+.External logging
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    # ...
+  zookeeper:
+    # ...
+  entityOperator:
+    # ...
+    topicOperator:
+      watchedNamespace: my-topic-namespace
+      reconciliationIntervalSeconds: 60
+      logging:
+        type: external
+        name: customConfigMap
+# ...
+----
+
+.Additional resources
+
+* Garbage collector (GC) logging can also be enabled (or disabled). For more information on GC, see xref:ref-jvm-options-deployment-configuration-kafka[]
```

Comment:
Suggested change: For more information about GC logging, see xref:ref-jvm-options-deployment-configuration-kafka[].

Reply:
Great idea!